config/filesystems.php

```diff
@@ -55,33 +55,18 @@
             'visibility' => 'public',
         ],
 
-//        's3' => [
-//            'driver' => 's3',
-//            'key' => env('AWS_ACCESS_KEY_ID'),
-//            'secret' => env('AWS_SECRET_ACCESS_KEY'),
-//            'region' => env('AWS_DEFAULT_REGION'),
-//            'bucket' => env('AWS_BUCKET'),
-//            'url' => env('AWS_URL'),
-//        ],
-
-        // https://github.com/Superbalist/laravel-google-cloud-storage
-        'gcs-public-static' => [
-            'driver' => 'gcs',
-            'project_id' => env('GOOGLE_CLOUD_PROJECT_ID', 'your-project-id'),
-            'key_file' => env('GOOGLE_CLOUD_STORAGE_KEY_FILE', null), // optional: /path/to/service-account.json
-            'bucket' => env('GOOGLE_CLOUD_STORAGE_BUCKET', 'your-bucket'),
-            'path_prefix' => env('GOOGLE_CLOUD_STORAGE_PATH_PREFIX', null), // optional: /default/path/to/apply/in/bucket
-            'storage_api_uri' => env('GOOGLE_CLOUD_STORAGE_API_URI', null), // see: Public URLs below
-            'visibility' => 'public', // optional: public|private
+        's3-static' => [
```

Comment:
Maybe having `s3` in the name here is a bad idea and this should rather be `static-assets` or sth?

Reply:
Yeah let's name it `static-assets`, I like that better as well.